changes/ocd-2973.sql

```diff
@@ -0,0 +1,1396 @@
+-- DROP and ADD constraint to ensure each criterion is only in the table once
+ALTER TABLE openchpl.certification_criterion_attribute DROP CONSTRAINT criterion_unique;
+ALTER TABLE openchpl.certification_criterion_attribute ADD CONSTRAINT criterion_unique UNIQUE (criterion_id);
+
+-- Add new column to indicate whether the criterion supports test tools
+ALTER TABLE openchpl.certification_criterion_attribute ADD COLUMN IF NOT EXISTS test_tool BOOL NOT NULL DEFAULT FALSE;
+
+-- Add the new test tools
+INSERT INTO openchpl.test_tool
+(name, last_modified_user)
+SELECT 'NCPDP Electronic Prescribing (eRx) Testing Tool', 1
```

Comment:
Is the last_modified_user the value you intended? I was thinking we normally use -1.